### ISSUE TITLE
Function calls in "forRoot()" causes "Function calls are not supported in decorators" error

### ISSUE THREAD

**Issue by hakimio** [tonysamperi/ngx-popperjs#5]
Calling [provideNgxPopperjsOptions() in forRoot() implementation](https://github.com/tonysamperi/ngx-popperjs/blob/master/projects/ngx-popperjs/src/lib/ngx-popperjs.module.ts#L40) causes "Function calls are not supported in decorators" error when compiling library with Angular 11.
It should be just changed to the following:
```ts
   public static forRoot(popperBaseOptions: NgxPopperjsOptions = {}): ModuleWithProviders<NgxPopperjsModule> {
        return {
            ngModule: NgxPopperjsModule,
            providers: [
                {provide: NGX_POPPERJS_DEFAULTS, useValue: popperBaseOptions},
            ]
        };
    }
```
Why do you need `provideNgxPopperjsOptions()` in the first place?

**Comment by tonysamperi** [comment]
Because the config extends defaults.

**Comment by hakimio** [comment]
It was released a long time ago.
https://github.com/angular/angular/blob/master/CHANGELOG.md#1120-2021-02-10
https://blog.angular.io/version-11-of-angular-now-available-74721b7952f7

**Comment by tonysamperi** [comment]
v11 of ngx-popperjs!
